src/classes/renderer/cache/simple/engine.vala

```diff
@@ -37,13 +37,16 @@ namespace pdfpc.Renderer.Cache {
          */
```

Comment:
See the alternative implementation [here](https://github.com/phillipberndt/pdfpc/commit/a4be87ab07a3c9e822f4214e35197704baf6ca42).